<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="homework">
       <attribute name="title">
        <string>作业</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QScrollArea" name="homeworkScroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="homeworkScrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>508</width>
             <height>665</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QToolBox" name="homeworkToolBox">
              <property name="styleSheet">
               <string notr="true">QToolBox QScrollArea { min-height: 400px; }</string>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <property name="tabSpacing">
               <number>6</number>
              </property>
              <widget class="QWidget" name="page">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>490</width>
                 <height>621</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <attribute name="label">
                <string>Page 1</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_3"/>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="newHomework">
             <property name="text">
              <string>新建</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="editHomework">
             <property name="text">
              <string>编辑</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteHomework">
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="topMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="finishedhomework">
       <attribute name="title">
        <string>已完成</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>508</width>
             <height>715</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QToolBox" name="finishedhomeworkToolBox">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="page_5">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>490</width>
                 <height>671</height>
                </rect>
               </property>
               <attribute name="label">
                <string>Page 2</string>
               </attribute>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="unfinishedhomework">
       <attribute name="title">
        <string>未完成</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>508</width>
             <height>715</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QToolBox" name="unfinishedhomeworkToolBox">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="page_7">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>490</width>
                 <height>671</height>
                </rect>
               </property>
               <attribute name="label">
                <string>Page 2</string>
               </attribute>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionsaveAndFlush"/>
    <addaction name="actionexit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionflush">
   <property name="text">
    <string>全部保存并刷新</string>
   </property>
  </action>
  <action name="actionconfirm1">
   <property name="text">
    <string>确认当前作业完成</string>
   </property>
  </action>
  <action name="actionconfirm2">
   <property name="text">
    <string>确认当前作业未完成</string>
   </property>
  </action>
  <action name="actionsaveAndFlush">
   <property name="text">
    <string>保存并刷新</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
